components/shared/embedder/lib.rs

```diff
@@ -121,13 +121,26 @@ pub enum ContextMenuResult {
 }
 
 #[derive(Deserialize, Serialize)]
-pub enum PromptDefinition {
-    /// Show a message.
-    Alert(String, IpcSender<()>),
-    /// Ask a Ok/Cancel question.
-    OkCancel(String, IpcSender<PromptResult>),
-    /// Ask the user to enter text.
-    Input(String, String, IpcSender<Option<String>>),
+pub enum ScriptDialog {
```

Comment:
just realised the spec has a name for these: [simple dialogs](https://html.spec.whatwg.org/multipage/#simple-dialogs). maybe we want to use that name?

Reply:
done in b969f42a252e42f69b1bd56239b73288ef1c9754